source/facter/3.0/release_notes.md

```diff
@@ -5,6 +5,103 @@ title: "Facter 3.0: Release Notes"
 
 This page documents the history of the Facter 3.0 series. (Previous version: [Facter 2.4 release notes](../2.4/release_notes.html).)
 
+Facter 3.0.2
+-----
+
+Released July 22, 2015.
+
+Shipped in puppet-agent version: 1.2.2.
+
+Facter 3.0.2 fixes several regressions in Facter 3.0.0 and implements new features to support legacy workflows.
+
+For JIRA issues related to Facter 3.0.2, see:
+- [Fixes for facter 3.0.2](https://tickets.puppetlabs.com/issues/?filter=15119)
+- [Introduced in facter 3.0.2](https://tickets.puppetlabs.com/issues/?filter=15118)
+
+### Feature: `--show-legacy` Flag Outputs Unstructured Facts
+
+Facter 3 outputs new, structured versions of many facts, such as `os`, and hides the equivalent unstructured legacy facts. In Facter 3.0.2, you can force Facter's output to include legacy facts by running it with the `--show-legacy` flag, providing a way to maintain legacy workflows while you convert them to the preferred `puppet facts`.
```

Comment:
Two thoughts here:
1) it's worth emphasizing that the hiding is CLI-only.
2) this only applies to the built-in facts so `puppet facts` is a separate concern

How about:

Facter 3 provides structured versions of many facts, such as `os`, and by default the command line output of facter hides the equivalent unstructured facts. Starting in facter 3.0.2, you can force Facter's output to include legacy facts by running it with the `--show-legacy` flag, providing a way to maintain legacy workflows while you convert to using structured facts.